src/utils/conversationCreator.js

```diff
@@ -5,9 +5,65 @@ const { CONVERSATION_MESSAGE_CONTEXT } = require('../models/conversations.model'
 const { getTransaction } = require('../blockchain/lib/web3Helpers');
 const {
   findSimilarDelegatedConversation,
+  findSimilarPayoutConversation,
   updateConversationPayments,
 } = require('../repositories/conversationRepository');
 
+async function addPaymentToExistingPayoutConversation({ payment, similarPayout, app }) {
```

Comment:
`PaymentExecuted` event can move money of multiple tokens to user wallet, so we should support multiple tokens. It would be better to create a helper function to use everywhere we record multiple payments in on field like withdraw, delegate and payout.

Reply:
@aminlatifi  https://github.com/Giveth/feathers-giveth/pull/487